cmd/tiller/tiller.go

```diff
@@ -65,14 +65,15 @@ const (
 	storageConfigMap = "configmap"
 	storageSecret    = "secret"
 
-	probeAddr = ":44135"
 	traceAddr = ":44136"
 
 	// defaultMaxHistory sets the maximum number of releases to 0: unlimited
 	defaultMaxHistory = 0
 )
 
 var (
+	probeAddr            = flag.String("probe-address", ":44135", "address:port to listen on for probe server")
+	probeEnable          = flag.Bool("probe", true, "enable probe server")
```

Comment:
Being able to disable probe server is a separate concern and should be in a separate commit. Do we even need the option?

Reply:
I am currently using this option. Using local tiller does not need to enable liveness and readiness probe.

Reply:
So technically the one that can be removed is probeAddr. Reason for having those was because the code change is based on this PR https://github.com/helm/helm/pull/3440

Reply:
They should still be separate commits. Only the enable flag commit would be fine.

Reply:
sure, I will remove the configuration part.